README.md

```diff
@@ -188,6 +189,8 @@ By Tommy Buonomo ([Frenchie Games](https://frenchiegames.app/#/))
 
 - Save The Potato [source-code](https://github.com/imaNNeo/save_the_potato) - 🥇 Winner of [Flame Game Jam 3.0](https://itch.io/jam/flame-jam-3/results) - Rotate the shields and save the potato from incoming orbs! By [imaNNeo](https://imanneo.dev). [for Android or iOS](http://savethepotato.com)
 
+- Flappy Dash - [Android](https://play.google.com/store/apps/details?id=games.pocket.flappydash.flappy_dash) - Gives classic flappy bird game play experience. By [Rutvik Panchal](https://twitter.com/rutvikpanchal_x)
```

Comment:
You can remove the entry from here and add the Play Store link in the open source section, similar to some of the other entries there.

Reply:
Updated, Please proceed.